docs/devel/go-code.md

```diff
@@ -53,6 +55,17 @@ and tips that our developers have found useful.
 - Consider making a separate Go tree for each project, which can make overlapping dependency management much easier. Remember to set the `$GOPATH` correctly! Consider [scripting](https://gist.github.com/timstclair/17ca792a20e0d83b06dddef7d77b1ea0) this.
 - Emacs users - setup [go-mode](https://github.com/dominikh/go-mode.el)
 
+## How to update the Go version
```

Comment:
It would be useful to say something about when would someone need/want to update the Go version.

This should also point to the place where we document the supported version:

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md#go-development-environment

That's enforced somewhere. Is it in one of the places you reference?